lang/rust-bootstrap/Portfile

```diff
@@ -46,13 +50,21 @@ if { ${subport} ne "${name}-transition" } {
     universal_variant       no
 }
 
-if { ${os.platform} eq "darwin" && [vercmp ${macosx_deployment_target} < 10.20] } {
+# support for ppc and ppc64 wold require
+#     bringing back PowerPC support in LLVM (https://releases.llvm.org/8.0.0/docs/ReleaseNotes.html)
+#     using GCC instead of LLVM as a backend (https://github.com/rust-lang/rustc_codegen_gcc)
+#     useinng an alternative Rust compiler
+#         GCC (https://rust-gcc.github.io)
+#         GCC (https://github.com/thepowersgang/mrustc)
+supported_archs             arm64 x86_64 i386
+if { ${os.platform} eq "darwin" && [vercmp ${macosx_deployment_target} < 11] } {
     # support for Apple silicon was added in macOS 11 (Darwin 20)
     # see https://trac.macports.org/ticket/65256
-    supported_archs         x86_64 i386 ppc ppc64
+    supported_archs         x86_64 i386
 }
 
-set py_ver                  3.11
+# see `def_v` in https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/python-1.0.tcl
+set py_ver                  3.12
```

Comment:
Has python 3.12 been set as the default Python for MacPorts yet?

Reply:
Yes, I believe the Python PG now defaults to Python 3.12 (9425f9bdddcba6541f493df996334b261a840f86).
If you look in the comment, there is a link to the Python PortGroup and the variable to look for (`def_v`).